docs-source/usersguide/library/02_compiling.rst

```diff
@@ -325,69 +325,50 @@ You should see a series of test results like this:
 can run them individually to get more detailed error information.  Note that
 some tests are stochastic, and therefore are expected to fail a small fraction
 of the time.  These tests will say so in the error message:
-::
 
-    ./TestReferenceLangevinIntegrator
+.. code-block:: none
 
     exception: Assertion failure at TestReferenceLangevinIntegrator.cpp:129.  Expected 9.97741,
         found 10.7884 (This test is stochastic and may occasionally fail)
 
-Congratulations! You successfully have built and installed OpenMM from source.
+Congratulations! You have successfully built and installed OpenMM from source.
+
 
 Building the Documentation (Optional)
 *************************************
 
-The documentation that you're currently reading, as well as the developer guide and API
-documentation can be built through CMake by setting the OpenMM option :code:`OPENMM_GENERATE_API_DOCS=ON`.
-
-User Guide and Developer Guide
-==============================
-
-Generating the user guide and developer guide requires the following dependencies
-
-* Sphinx (http://sphinx-doc.org/)
-
-* sphinxcontrib-bibtex (https://pypi.python.org/pypi/sphinxcontrib-bibtex)
-
-These dependencies may not be available in your system package manager, but should
-be installable through Python's ``pip`` package manager. ::
-
-   pip install sphinx sphinxcontrib-bibtex
-
-The developer and user guides can be built either as HTML or a PDFs. Building the
-PDF version will also require a functional LaTeX installation.
-
-To build the HTML version of the documentation, type: ::
-
-  make sphinxhtml
-
-To build the PDF version of the documentation, type: ::
-
-  make sphinxpdf
-
-
-Python and C++ API Documentation
-================================
+The documentation that you're currently reading, as well as the Developer Guide and API
+documentation, can be built through CMake.  To do that, you need to install a few
+additional tools.  The easiest way is to use :code:`pip` to install them into
+your Python environment.  The following command installs everything needed to
+build documentation in HTML format.
+::
 
-The following dependencies are required to build the Python and C++ API documentation.
+    pip install sphinx sphinxcontrib-bibtex sphinxcontrib-lunrsearch sphinxcontrib-autodoc_doxygen jinja2
```

Comment:
I'm not sure how I feel about suggesting users install things with pip that can be installed with conda. It's probably not the end of the world but we should at least make a note somewhere to change this to conda once the pip-only deps are no longer required (which I'm working on now), and I would be in favour of having two lines here so that the dependencies available on conda-forge are installed with it.

Reply:
I'll be happy to switch the instructions to using conda as soon as the packages are available on conda! :)